.changeset/plain-animals-change.md

```diff
@@ -0,0 +1,5 @@
+---
+"ponder": patch
+---
+
+Introduced `eth_getLogsRangeLimit` to chains config. [Read more](http://localhost:5173/docs/config/chains#eth_getlogs-range-limit).
```

Comment:
Maybe `ethGetLogsBlockRange` ? It's not really a "range limit", it's the value that will be used for every request.